sodium/test/crypto_box_test.pony

```diff
@@ -38,7 +38,25 @@ class CryptoBoxTest is UnitTest
     try CryptoBox.open(crypt', nonce', csk, bpk)
       h.fail("Cyril shouldn't be able to open Bob's message to Alice.")
     end
-    
+
+    ///
+    // Key pairs generated with seeds
+
+    (let sks, let pks) = CryptoBox.seed_keypair(CryptoBoxSeed("Hello seeds!"))
```

Comment:
Yes, it is probably best that the user specifies the full 32 bytes. How are strings treated in Pony? Coming from Javascript, where Strings are constants, you'd use a Buffer / Uint8Array here as that gives you access to the underlying memory, and allow you to zero out the buffer when you're done using it, as to prevent illicit reads. Are strings safe in this regard in Pony? I guess the threat model of lingering strings is a bit weird, but at least you can guarantee when a sensitive value is in memory somewhere, and when it is cleared.